libpod/sqlite_state_internal.go

```diff
@@ -0,0 +1,288 @@
+package libpod
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/podman/v4/libpod/define"
+	"github.com/sirupsen/logrus"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// Initialize all required tables for the SQLite state
+func sqliteInitTables(conn *sql.DB) (defErr error) {
+	// Technically we could split the "CREATE TABLE IF NOT EXISTS" and ");"
+	// bits off each command and add them in the for loop where we actually
+	// run the SQL, but that seems unnecessary.
+	const dbConfig = `
+        CREATE TABLE IF NOT EXISTS DBConfig(
+                Id            INTEGER PRIMARY KEY NOT NULL,
+                SchemaVersion INTEGER NOT NULL,
+                Os            TEXT    NOT NULL,
+                StaticDir     TEXT    NOT NULL,
+                TmpDir        TEXT    NOT NULL,
+                GraphRoot     TEXT    NOT NULL,
+                RunRoot       TEXT    NOT NULL,
+                GraphDriver   TEXT    NOT NULL,
+                VolumeDir     TEXT    NOT NULL,
+                CHECK (Id IN (1))
+        );`
+
+	const idNamespace = `
+        CREATE TABLE IF NOT EXISTS IDNamespace(
+                Id TEXT PRIMARY KEY NOT NULL
+        );`
+
+	const containerConfig = `
+        CREATE TABLE IF NOT EXISTS ContainerConfig(
+                Id              TEXT    PRIMARY KEY NOT NULL,
+                Name            TEXT    UNIQUE NOT NULL,
+                PodID           TEXT,
```

Comment:
Can't the `PodID` be `ID`?

Same for other tables below.

Reply:
No? We already have a ID field for the container ID. PodID needs to be a separate column.